public/wp-content/themes/ColoredCow/functions.php

```diff
@@ -26,9 +35,8 @@ function add_subscriber(){
     $post_email = $_POST['email'];
     $post_password= $_POST['password'];
     $post_gender= $_POST['gender'];
-
     $hash = wp_hash_password( $post_password );
-    if(check_duplicate_entry($post_phone,$post_email)==true){
+    if(check_duplicate_entry($post_event_id,$post_phone,$post_email)==true){
```

Comment:
do we need to call the function with event id? Refer to the corresponding review comment.